pkg/apis/v1alpha1/register.go

```diff
@@ -55,23 +55,48 @@ var (
 	AMIFamilyBottlerocket = "Bottlerocket"
 	AMIFamilyAL2          = "AL2"
 	AMIFamilyUbuntu       = "Ubuntu"
+	AMIFamilyWindows2019  = "Windows2019"
+	AMIFamilyWindows2022  = "Windows2022"
 	AMIFamilyCustom       = "Custom"
 	SupportedAMIFamilies  = []string{
 		AMIFamilyBottlerocket,
 		AMIFamilyAL2,
 		AMIFamilyUbuntu,
+		AMIFamilyWindows2019,
+		AMIFamilyWindows2022,
 		AMIFamilyCustom,
 	}
 	SupportedContainerRuntimesByAMIFamily = map[string]sets.String{
 		AMIFamilyBottlerocket: sets.NewString("containerd"),
 		AMIFamilyAL2:          sets.NewString("dockerd", "containerd"),
 		AMIFamilyUbuntu:       sets.NewString("dockerd", "containerd"),
+		AMIFamilyWindows2019:  sets.NewString("containerd"),
```

Comment:
On Windows, similar to AL2, both `dockerd` and `containerd` runtimes are supported but the same are based on the EKS version.

As discussed with @jonathan-innis, the container runtime validation is obsolete code. Even so, Should we make Windows consistent with AL2?

Reply:
Yes, I think let's pull it out or do the validation in a separate PR. @topikachu We should have both container runtimes supported here for both windows versions.

Reply:
Fixed.
Add both dockerd and containerd